server/api_server/api_server_start.py

```
@@ -469,27 +710,37 @@ def api_devices_search():
 @app.route('/mcp/sse/devices/latest', methods=['GET'])
 @app.route('/devices/latest', methods=['GET'])
-def api_devices_latest():
+@validate_request(
+    operation_id="get_latest_device",
+    summary="Get Latest Device",
+    description="Get information about the most recently seen/discovered device.",
+    response_model=GetDeviceResponse,
+    tags=["devices"],
+    auth_callable=is_authorized
+)
+def api_devices_latest(payload=None):
```

Ruff (0.14.11)

Unused function argument: `payload`

(ARG001)